### DISCUSSION TITLE
WaitForInitialLoadAsync

### DISCUSSION THREAD

**Discussion by Mephisto090** [cefsharp/CefSharp.Dom#24] [Q&A]
How can I query the status of `WaitForInitialLoadAsync`?

Example: loading, finished loading, etc.

**Comment by amaitland** [comment]
Please clarify what you are asking.

**Reply by Mephisto090** [reply]
`WaitForInitialLoadAsync` lets the program wait until the page has been fully loaded. Can I somehow query the status in which he is right now?

**Reply by valcriss** [reply]
You  can use the ChromiumWebBrowser IsLoading property ?
